.travis.yml

```diff
@@ -65,6 +65,8 @@ env:
     - "TESTLANG=Nim"
     - 'TESTDIR="Nim/httpbeast Nim/jester Nim/basolato"'
     - "TESTLANG=OCaml"
+    - "TESTDIR=OCaml/morph"
```

Comment:
hey @ulrikstrid explicitly listing the testdir does not seem to be necessary - for example https://github.com/TechEmpower/FrameworkBenchmarks/blob/30bb2a4047fd24475392f746faef03133e0c5c2e/.travis.yml#L15-L16

(my PR for opium was also merged yesterday, so the list is not full)